### PR TITLE
added common easy responses for ARR content

### PR DESCRIPTION
**Pull request by Akurosia**

### CODE REVIEW

File: ui/raidboss/data/02-arr/raid/t11.js [hunk 1]
```diff
@@ -69,14 +69,7 @@
       regexJa: Regexes.ability({ id: 'B76', source: 'カーリア', capture: false }),
       regexCn: Regexes.ability({ id: 'B76', source: '卡利亚', capture: false }),
       regexKo: Regexes.ability({ id: 'B76', source: '칼리야', capture: false }),
-      infoText: function(data) {
-        if (!data.firstSeed)
```
Comment (quisquous) [inline ui/raidboss/data/02-arr/raid/t11.js]: What about this logic?

File: ui/raidboss/data/02-arr/raid/t11.js [hunk 2]
```diff
@@ -89,14 +82,7 @@
       regexJa: Regexes.ability({ id: 'B77', source: 'カーリア', capture: false }),
       regexCn: Regexes.ability({ id: 'B77', source: '卡利亚', capture: false }),
       regexKo: Regexes.ability({ id: 'B77', source: '칼리야', capture: false }),
-      infoText: function(data) {
-        if (!data.firstSeed)
```
Comment (quisquous) [inline ui/raidboss/data/02-arr/raid/t11.js]: This too.

File: ui/raidboss/data/02-arr/raid/t11.js [hunk 3]
```diff
@@ -110,10 +96,7 @@
       regexCn: Regexes.hasHP({ name: '卡利亚', hp: '60', capture: false }),
       regexKo: Regexes.hasHP({ name: '칼리야', hp: '60', capture: false }),
       sound: 'Long',
-      infoText: {
-        en: 'Out of Middle',
-        fr: 'En dehors du centre',
-      },
+      response: Responses.getOut(),
```
Comment (quisquous) [inline ui/raidboss/data/02-arr/raid/t11.js]: This is not the same.  Get out is ambiguous and usually means away from the boss.  This means don't stand in the center of the arena.

File: ui/raidboss/data/02-arr/raid/t6.js [hunk 4]
```diff
@@ -169,10 +166,7 @@
       regexJa: Regexes.startsUsing({ id: '86C', source: 'ラフレシア', capture: false }),
       regexCn: Regexes.startsUsing({ id: '86C', source: '大王花', capture: false }),
       regexKo: Regexes.startsUsing({ id: '86C', source: '라플레시아', capture: false }),
-      infoText: {
-        en: 'Stack for Acid',
-        fr: 'Packez vous pour Acide',
-      },
+      response: Responses.stack(),
```
Comment (quisquous) [inline ui/raidboss/data/02-arr/raid/t6.js]: I would leave this.  Maybe I'm overthinking, but I think it's nice to differentiate stack (shared damage marker) with bait (stack up to drop ground aoes)? "Stack for acid" at least indicates why?

File: ui/raidboss/data/02-arr/raid/t8.js [hunk 5]
```diff
@@ -7,18 +7,7 @@
...
-            en: 'Laser Stack on YOU',
-            fr: 'Package laser sur VOUS',
-          };
-        }
-        return {
-          en: 'Stack on ' + data.ShortName(matches.target),
-          fr: 'Package sur ' + data.ShortName(matches.target),
-        };
-      },
+      response: Responses.stackOn(),
```
Comment (quisquous) [inline ui/raidboss/data/02-arr/raid/t8.js]: stackOn('info') to match previous

File: ui/raidboss/data/02-arr/raid/t6.js (hunk 4 above)
Comment (Akurosia) [reply ui/raidboss/data/02-arr/raid/t6.js]: ok i will revert it

**Pull request merged**